app/serializers/event_serializer.rb

```diff
@@ -9,9 +9,10 @@ class EventSerializer < ActiveModel::Serializer
   attribute :event_name, key: :name
   attribute :service_description, key: :service
   attributes :estimated_distance, :exception_note, :event_details
-  attribute :agency_name
+  attributes :agency_name
```

Comment:
The importance of an s? Is it...:
- attribute : oldname, key: :new_name?
- attributes: enumerate the specific fields you want in the model?

Reply:
This is left over junk from when the display_age_adult and display_age_senior was included in the event_serializer.  I am surprised linter didn't complain about this.  I will remove the attributes :agency_name line.  attribute : is for one attribute, attributes : is for more than one.  Good catch!